fluster/image/raw_video.py

```diff
@@ -0,0 +1,104 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2021, Fluendo, S.A.
+#  Author: Manuel Jimeno Martínez <mjimeno@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+from pathlib import Path
+from typing import Optional, List
+
+from fluster.image.raw_buffer import RawBuffer
+from fluster.image.raw_image import RawImage, get_meta_from_name
+
+
+class RawVideo:
+    """Class to handle the raw video"""
+
+    def __init__(
+        self,
+        buffer: bytes,
+        video_name: str,
+        width: int,
+        height: int,
+        buffer_type: RawBuffer.Type,
+    ):
+        self._buffer: bytes = buffer
+        self._video_name: str = video_name
+        self._width: int = width
+        self._height: int = height
+        self._buffer_type: RawBuffer.Type = buffer_type
+        self._frames = []
+
+        if buffer_type == RawBuffer.Type.YUV420:
+            if (width % 4) or (height % 4):
```

Comment:
Be clear (also parenthesis not needed): `if width % 4 != 0 or height % 4 != 0`.

Reply:
done